vision/google/cloud/vision/image.py

```diff
@@ -182,17 +182,28 @@ def detect_faces(self, limit=10):
         annotations = self.detect(features)
         return annotations[0].faces
 
-    def detect_full_text(self, limit=10):
+    def detect_full_text(self, language_hints=None, limit=10):
         """Detect a full document's text.
 
+        :type language_hints: list
+        :param language_hints: (Optional) A list of BCP-47 language codes. See:
+                               https://cloud.google.com/vision/docs/languages
+
         :type limit: int
-        :param limit: The number of documents to detect.
+        :param limit: (Optional) The number of documents to detect.
 
         :rtype: list
         :returns: List of :class:`~google.cloud.vision.text.TextAnnotation`.
         """
-        features = [Feature(FeatureTypes.DOCUMENT_TEXT_DETECTION, limit)]
-        annotations = self.detect(features)
+        feature_type = image_annotator_pb2.Feature.DOCUMENT_TEXT_DETECTION
```

Comment:
I was using `google.cloud.vision.feature.Feature` and `image_annotator_pb2.AnnotateImageRequest` requires `image_annotator_pb2.Feature`. Moving forward I think we should use the protos more.